pull-request.sh

```diff
@@ -106,6 +120,15 @@ main () {
         PULL_REQUEST_DRAFT="true"
     fi
 
+    if [ -z "${PULL_REQUEST_MAINTAINER_CAN_MODIFY}" ]; then
+        echo "No explicit preference for maintainers can modify PR: created PRs with maintainers can modify=true."
```

Comment:
The variable names don't make sense - by not setting PULL_REQUEST_MAINTAINER_CAN_MODIFY we set the variable to true, which is the opposite of what I'd expect. It's more intuitive (if the maintainer by default can modify) to set a variable like `PULL_REQUEST_MAINTAINER_CANNOT_MODIFY`. This should also be reflected in the message - if I don't set anything and read this message, I don't know what I'd set to get the opposite behavior.